scripts/synthdnn/synthdnn.py

```diff
@@ -165,7 +165,7 @@ def setup_matmul_subparser(subparsers):
         "-t",
         "--types",
         default="all",
-        help='dt:dt:dt(optional fpmath-mode), comma separated list of type configurations or "all" for every supported type',
+        help='dt:dt:dt(optional fpmath-mode), comma separated list of type configurations. "%%N" will match the Nth given data type. Giving a single data type instead of 3 will match any supported configurations using that type.',
```

Comment:
It seems to me a documentation page on the syntax should be updated/added as well. It would be good to explain new capabilities with examples of command line syntax and the outcome.

Reply:
I updated the readme with a more verbose description of the options, as well as a few examples. I also removed a shortcut syntax we had (generation and collection in one command) to simplify the syntax. Let me know if you'd like any changes to the readme, please.

Reply:
Thanks for taking care of it. Definitely looks better.